opcua/server/internal_subscription.py

```diff
@@ -140,10 +142,12 @@ def _delete_monitored_items(self, mid):
 
     def datachange_callback(self, handle, value, error=None):
         if error:
-            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'", self, handle, error)
+            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'",
```

Comment:
I have never been able to decide anything about line length. I usually do not care but those lines where to long for my screen. I am open to suggestions